src/box/lua/net_box.lua

```diff
@@ -1059,15 +1095,15 @@ this_module.self = {
         end
         local result
         if obj ~= nil then
-            return handle_eval_result(pcall(proc, obj, ...))
+            return handle_eval_result(pcall(proc, obj, unpack(args)))
```

Comment:
Please change the signature of handle_eval_result() to avoid unpack. We may go one step forward and change the return  value convention as well.